### PR TITLE
Remove - unused FormPostHelper code

### PR DESCRIPTION
**Pull request by lmarceau**
## :scroll: Tickets
No ticket

## :bulb: Description
While checking some code for the WebEngine, stumbled on this FormPostHelper which doesn't seem to be used anymore unless I missed something?

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Comment by nbhasin2** [comment]
Let me double check before we merge it.

**Review by nbhasin2 — APPROVED** [review]
We can remove this cc @issammani

**Comment by issammani** [comment]
> We can remove this cc @issammani

Yes it's safe to remove this. We don't need it anymore. 

TL;DR
Webkit had [a bug](https://bugs.webkit.org/show_bug.cgi?id=167131) where it was not possible to access a post request body. `FormPostHelper.swift` and [`FormPostHelper.js`](https://github.com/mozilla-mobile/firefox-ios/pull/3434/commits/64cd9c843f22d9a542d9ce060caac287d34fe40d#diff-c15308e5e381c5069f2a380da6bda8aa9bc226df7c7a6e8dea8c56e79930dd04) were added as a workaround. Basically to get the form data from the page itself and send it to swift. The bug has been fixed since, so this is not needed anymore 😄
